pkg/webhook/parser.go

```diff
@@ -14,223 +14,155 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Package webhook contains libraries for generating webhookconfig manifests
+// from markers in Go source files.
+//
+// The markers take the form:
+//
+//  +kubebuilder:rbac:failurePolicy=<string>,groups=<[]string>,resources=<[]string>,verbs=<[]string>,versions=<[]string>,name=<string>,path=<string>,mutating=<bool>
```

Comment:
It seems it now requires all arguments to be in the same line. We need to remember to update the book.